app/assets/stylesheets/players.sass

```diff
@@ -1,3 +1,19 @@
 .players
   .manual_seed
     width: 80px
+
+.identities_form
+  .awesomplete
+    display: block
+  > *
```

Comment:
i don't know sass so i will trust you about the formatting here.  :D The indentation and the > * looks a bit inconsistent, but you tell me if that is fine or not.

Reply:
I was being lazy. `> *` will select all the immediate children of `.identities_form`. I'm changing it to select the two different input containers so it's more clear.